server/routerlicious/packages/lambdas/src/moira/README.md

```diff
@@ -0,0 +1 @@
+Test service to submit messages to MH
```

Comment:
We should have a better description here. Like what the lambda do, how it knows about property set dds ops, what service it talks to....